### PR TITLE
Feature/wdp190703 24/section promotion markup and style

### PR DESCRIPTION
**Pull request by MiroslawChmielecki**
I created section "promotion" , added few files png in IMAGES and added styles !!

### CODE REVIEW

File: src/sass/components/_section--promotion.scss [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    margin-bottom: 30px;
+
+    .inbox-item-left {
+      height: 100%;
+      width: 35%;
+      background-image: url("../images/redChairNice.png");
+      background-size: 190px;
+      background-position: 180px 9px;
+    }
+    .inbox-item-center {
```
Comment (thisisui) [inline src/sass/components/_section--promotion.scss]: enter above

File: src/sass/components/_section--promotion.scss [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+      width: 40%;
+      z-index: 10;
+      padding: 0 20px;
+      min-width: 120px;
+
+      .box-two-first-line {
+        font-size: 25px;
+        padding-bottom: 0;
+        margin: 0;
+        .bold-word {
```
Comment (thisisui) [inline src/sass/components/_section--promotion.scss]: enter above

File: src/sass/_variables.scss [hunk 3]
```diff
@@ -11,15 +11,17 @@ $text-color: #2a2a2a;
 
 $form-border-color: #292929;
 
+$promotion-box-overlay-color: rgba(42, 42, 42, 0.4);
+$promotion-smaller-box-above-color: #e8edf7;
```
Comment (thisisui) [inline src/sass/_variables.scss]: I would not create names of colors with direct functionality of that color - more like promotion-primary-color, promotion-secondary-color - try to make names more generic - less reformatting if client will change his mind in the future

File: src/sass/components/_section--promotion.scss (hunk 1 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_section--promotion.scss]: fixed !!

File: src/sass/components/_section--promotion.scss (hunk 2 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_section--promotion.scss]: fixed !!

File: src/sass/_variables.scss (hunk 3 above)
Comment (MiroslawChmielecki) [reply src/sass/_variables.scss]: Variables renamed and used in code !!

File: src/partials/40_section-promotion.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+              <p class="box-two-third-line">$200.00</p>
+            </div>
+            <div class="inbox-item inbox-item-right"></div>
+          </div>
+
+          <div class="smaller-box-below fit-flex">
+            <div class="inbox-item-small inbox-item-small-left"></div>
+            <div class="inbox-item-small">
+              <p class="box-three-first-line">
+                <span class="bold-word">SPECIAL</span> COLLECTION
```
Comment (matwjablonski) [inline src/partials/40_section-promotion.html]: uppercase create using css

Comment (MiroslawChmielecki) [reply src/partials/40_section-promotion.html]: text-transform: uppercase used in styles !!

Comment (MiroslawChmielecki) [reply src/partials/40_section-promotion.html]: fixed !

Comment (DzesikaTkaczyk) [reply src/partials/40_section-promotion.html]: Jakbyś jeszcze dodał temu special collection mały padding z prawej to by optycznie wyglądało lepiej

Comment (MiroslawChmielecki) [reply src/partials/40_section-promotion.html]: ok według sugestii zmieniłem padding z 10px na 17px, tylko mialem juz zatwierdzone zadanie przez thisisui :/

**Pull request merged**